src/components/research/page.tsx

```diff
@@ -0,0 +1,11 @@
+const ResearchImage = () => {
+  return (
+    <>
+      <div>
+        <img src="/research/rectangle8.png" />
```

Comment:
```suggestion
        <img className="w-screen aspect-[5/1]" src="/research/rectangle8.png" />
```

style the img so it matches the relative size on the figma